### ISSUE TITLE
change the "visualEffectState" option at runtime

### ISSUE THREAD

**Issue by EYHN** [electron/electron#25513]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

### Problem Description
Since #24471 #25083 , `BrowserWindow` adds an option "visualEffectState" that allows us to customize the app visual effect state be inactive or active regardless of window activation state. 

Just like we have [BrowserWindow.setVibrancy(type)](https://www.electronjs.org/docs/api/browser-window#winsetvibrancytype-macos) which can change the vibrancy effect of the application at runtime.

I hope to have a "BrowserWindow.setVisualEffectState(state)" method to change the "visualEffectState" option at runtime.

### Additional Information

https://github.com/EYHN/vscode-vibrancy/issues/67

**Comment by eatinsnacks** [comment]
I could benefit from this too.

**Comment by eatinsnacks** [comment]
Still interested in this.

**Comment by AurevoirXavier** [comment]
> Still interested in this.

+1

**Comment by Dessader** [comment]
+1

**Comment by djklu31** [comment]
+1

**Comment by foxt** [comment]
Yeah, it doesn't make sense to have there be a disabled option if you can only apply it at init time.

**Comment by mayurankv** [comment]
I'd also love this!
